### PR TITLE
Change Java Util Logging to SLF4J and add SLF4J simple logger for tests

### PR DESCRIPTION
**Pull request by mprus**
Created for #184

### CODE REVIEW

**Comment by domesticmouse** [comment]
Can you rebase this PR?

File: build.gradle [hunk 1]
```diff
@@ -47,11 +47,13 @@ dependencies {
     compile 'com.squareup.okhttp:okhttp:2.7.5'
     compile 'joda-time:joda-time:2.4'
     compileOnly 'com.google.appengine:appengine-api-1.0-sdk:1.9.36'
+    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.21'
```
Comment (domesticmouse) [inline build.gradle]: Can you please make this the same as the previous compile lines?

File: src/main/java/com/google/maps/GeoApiContext.java [hunk 2]
```diff
@@ -15,23 +15,27 @@
 
 package com.google.maps;
 
+import com.google.apphosting.api.search.DocumentPb;
```
Comment (domesticmouse) [inline src/main/java/com/google/maps/GeoApiContext.java]: Where did this import come from? I don't think we want this in GeoApiContext.

File: src/main/java/com/google/maps/internal/OkHttpPendingResult.java [hunk 3]
```diff
@@ -53,7 +55,7 @@
 import java.util.List;
 import java.util.concurrent.ArrayBlockingQueue;
 import java.util.concurrent.BlockingQueue;
-import java.util.logging.Logger;
+
```
Comment (domesticmouse) [inline src/main/java/com/google/maps/internal/OkHttpPendingResult.java]: Drop extra blank line

**Comment by hth** [comment]
@mprus I have made necessary changes. Can you please add me as a collaborator to your `google-maps-services-java` and I can push these changes to so that we can get this out.

**Comment by hth** [comment]
@domesticmouse @mprus I have resolved conflicts. Please take a look.

**Comment by domesticmouse** [comment]
@hth can you please sign the CLA? I can't accept the PR without CLA signoff. Thanks!

**Comment by hth** [comment]
@domesticmouse Signed CLA.

**Comment by hth** [comment]
@domesticmouse @mprus Thanks!

**Pull request merged**